### PR TITLE
[Refactor] Refactor text detection config

### PR DESCRIPTION
**Pull request by cuhk-hbsun**

### CODE REVIEW

File: tests/test_apis/test_model_inference.py [hunk 1]
```diff
@@ -11,7 +12,13 @@
 
 def build_model(config_file):
     device = 'cpu'
-    model = init_detector(config_file, checkpoint=None, device=device)
+
+    cfg = Config.fromfile(config_file)
+    if cfg.model.get('backbone', False):
+        if cfg.model.backbone.type == 'mmdet.ResNet':
+            cfg.model.backbone.norm_cfg = dict(type='BN', requires_grad=True)
```
Comment (gaotongxiao) [inline tests/test_apis/test_model_inference.py:19]: Why

Comment (cuhk-hbsun) [reply tests/test_apis/test_model_inference.py:19]: `SynthBN` cannot have a training on one GPU.

Comment (cuhk-hbsun) [reply tests/test_apis/test_model_inference.py:19]:
```python
if cfg.model.get('backbone', False):
        if cfg.model.backbone.type == 'mmdet.ResNet':
            if cfg.model.backbone.get('norm_cfg', False):
                if cfg.model.backbone.norm_cfg.type == 'SyncBN':
                    cfg.model.backbone.norm_cfg.type = 'BN'
```
Will this be better?

Comment (cuhk-hbsun) [reply tests/test_apis/test_model_inference.py:19]: use `model = revert_sync_batchnorm(model)` instead.

**Review by gaotongxiao — APPROVED** [review]

**Review by innerlee — APPROVED** [review]

**Pull request merged**